be/src/vec/functions/function_binary_arithmetic.h

```diff
@@ -449,6 +469,18 @@ struct DecimalBinaryOperation {
 private:
     /// there's implicit type convertion here
     static NativeResultType apply(NativeResultType a, NativeResultType b) {
+        if (config::enable_execution_decimalv3) {
+            if constexpr (OpTraits::can_overflow && check_overflow) {
+                NativeResultType res;
+                if (Op::template apply<NativeResultType>(a, b, res)) {
+                    LOG(FATAL) << "Decimal math overflow";
```

Comment:
Better not use FATAL log, it will cause BE crash.
Could we return a invalid value or null to indicate this error?
And in what situation can we get here?

Reply:
Decimal addition and multiplication may overflow.

Reply:
how about to return null when overflow?

Reply:
The column may not be ColumnNullable, so we can't return null.